metricbeat/module/system/_meta/config.yml

```diff
@@ -14,6 +14,7 @@
   process.include_top_n:
     by_cpu: 5      # include top 5 processes by CPU
     by_memory: 5   # include top 5 processes by memory
+  #diskio.include_devices: []
```

Comment:
Can you remove this line? We should not have this in our short config. Also run  `make update` afterwards.